readme/configuration-management.md

```diff
@@ -42,6 +52,45 @@ The solution is to make sure that the referenced content exists before the featu
 * Use the default_content module to export the referenced content as JSON files, and store these files with your feature or in a dependency.
 * Use Migrate and a custom module to create default content from any number of custom sources, such as JSON files stored with your feature.
 
+### Updating core and contributed modules
+
+Caution must be taken when updating core and contributed modules. If those updates make changes to a module’s configuration or schema, you must make sure to also update your exported configurations. Otherwise, the next time you run updates it will import a stale configuration schema and cause unexpected behavior.
+
+The best way to handle this is to always follow these steps when updating contributed and core modules:
+
+1. Start from a clean local setup or refresh. If you are using Features, ensure that there are no overridden configuration. The `cm.features.no-overrides` flag in [project.yml](https://github.com/acquia/blt/blob/8.x/template/blt/project.yml#L62) can assist with this by halting builds with overridden features.
+2. Use `composer update` to download the new module version(s).
```

Comment:
Do we want to recommend plain `composer update` (which can have a lot of unintended consequences), or more specific `composer update drupal/modulename --with-dependencies` (which seems to be more foolproof for someone who isn't meticulous about monitoring what happens to `composer.lock`).